pvm/ts/src/engine/texasHoldem2.test.ts

```
@@ -134,7 +134,12 @@ describe.only("Texas Holdem Game", () => {
             game.join2("0x980b8D8A16f5891F41871d878a479d81Da52334c", 1000000000000000000000n);
         });
 
-        it.only("should have player status set to active", () => {
+        it.only("should have correct table properties", () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)